src/util/validationSchema.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod'
+
```

Comment:
since we wil have many schemas in the project I suggest creating a folder called `schema` in `src` and then put the schemas there.

The folder would have a file structure like 

```
/src
  /schema
    /index.ts
    /login.ts
    /signup.ts